planning/behavior_path_planner/src/scene_module/lane_change/normal.cpp

```diff
@@ -482,6 +482,141 @@ PathWithLaneId NormalLaneChange::getPrepareSegment(
   return prepare_segment;
 }
 
+LaneChangeTargetObjects NormalLaneChange::getTargetObjects(
+  const PredictedObjects & objects, const lanelet::ConstLanelets & current_lanes,
+  const lanelet::ConstLanelets & target_lanes,
+  const lanelet::ConstLanelets & target_backward_lanes) const
```

Comment:
If you want to reduce further,I would also recommend `target_backward_lanes` to be computed in this function.

Reply:
done with https://github.com/autowarefoundation/autoware.universe/pull/4273/commits/82c4c8c2787bd902aef7d3e28693971e1f6a4935